### PR TITLE
November Newsletter

### PR DESCRIPTION
**Pull request by permcody**

### CODE REVIEW

**Comment by permcody** [comment]
@idaholab/moose-team - I'm starting the newsletter since @aeslaughter is currently on travel. I wanted everyone to get a chance to contribute before we get too close to Thanksgiving. Also, I'll be out next week, so I wanted my changes in...

File: modules/doc/content/newsletter/2019_11.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+store is that you don't have to treat your normal simulation and [recover](restart_recover.md)
+simulations any differently from one and other. You'll be able to retrieve properties the same
+way in either case simplifying your startup code. This makes a lot of difference if you are
+using the MeshGeneration system to build your mesh programatically. The MetaData store is
+populated very early on recover runs so the data is available during the execution of any
+custom Actions that you might create.
+
+## libpng detection test has moved to MOOSE's configure
+
+Does anyone remember when MOOSE got a proper [configure system](build_system.md)? No? Good,
```
Comment (lindsayad) [inline modules/doc/content/newsletter/2019_11.md]: Haha I guess I really didn't add anything to previous newsletters about it...

**Comment by permcody** [comment]
@idaholab/moose-team - Anything else?

**Comment by fdkong** [comment]
> @idaholab/moose-team - Anything else?

I will add my contributions today.

**Comment by lindsayad** [comment]
I like that @fdkong somehow failed a threaded test in porous flow... That seems problematic

**Comment by permcody** [comment]
@lindsayad - but @fdkong didn't do anything!

**Comment by lindsayad** [comment]
Haha he really didn't this time...

**Review by lindsayad — APPROVED** [review]

**Pull request merged**